lib/api/helpers/ordering.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module API
+  module Helpers
+    module Ordering
```

Comment:
Could you add documentation? Even if it's a bit obvious, I believe it's good practice to write what it's expected to happen when you include this module, or you use this method (e.g. the parameters that are going to be used)

Reply:
Sure.

Reply:
Done.